### ISSUE TITLE
Document appending of plots

### ISSUE THREAD

**Issue by James-G-Hill** [Davide-sd/sympy-plot-backends#10]
Hello, I'm not sure if this is a `sympy-plot-backends` feature, a `sympy` feature or maybe a `matplotlib` feature, but there doesn't seem to be anything in the documents showing how charts can be appended to each other in `sympy-plot-backends`.

I did the following, remembering how I previously used this with the `sympy` plotting modules:

```
p1 = plot3d_spherical(1, show=False)
p2 = plot3d_list(([0, 1, 2], [0, 2, 1], [1, 0, 2]), show=False)
p1.append(p2[0])
p1.show()
```

This showed me a chart combining the two I had created as desired. I could find it at the following link, but not in the documentation for this package:
https://docs.sympy.org/latest/modules/plotting.html#sympy.plotting.plot.Plot.append

Maybe it's not appropriate or maybe an oversight? Just thought I'd mention it because it would seem to be useful information to have.

Absolutely love this package by the way!

**Comment by Davide-sd** [comment]
It is (and was) documented [here](https://sympy-plot-backends.readthedocs.io/en/latest/modules/backends/matplotlib.html#spb.backends.matplotlib.MatplotlibBackend.append).

I never really used `append` or `extend` as they usually requires more typing. That's why I've implemented the addition of plot objects. There are some examples scattered around the documentation of adding two or more plots together.

Anyway, I have created a [new tutorial](https://sympy-plot-backends.readthedocs.io/en/latest/tutorials/tut-1.html) illustratinga couple of ways to combine plots. Hope that will help. 

> Absolutely love this package by the way!

Thanks for the kind word! :)
